cmd/main.go

```diff
@@ -60,6 +61,11 @@ func init() {
 	if err := viper.BindPFlag("refresh_interval", flags.Lookup("refresh-interval")); err != nil {
 		log.Fatal(err)
 	}
+
+	flags.Float64("batch-size-percent", 1, "loadCerts batch size percentage")
```

Comment:
I don't think it's clear from the help text if this should be specified as a percentage from `0.0 - 1.0` or `0 - 100` or if it should or shouldn't have a `%` trailing after it.

What do you think about clarifying that?

Reply:
good call! 7c0e9b4848b9f8e4e5d9149bc1f0e731c4b36112